ui/packages/catalog-ui-search/src/main/webapp/component/query-time/query-time.view.js

```diff
@@ -208,15 +208,17 @@ module.exports = Marionette.LayoutView.extend({
                 value: [currentBefore],
                 id: 'Before',
                 placeholder: 'Limit search to before this time.',
-                type: 'DATE'
+                type: 'DATE',
```

Comment:
You can modify the id directly here since we aren't using it for identification later.  Basically, update the ids to be what the labels are and remove the labels that were added.